include/fuse.h

```diff
@@ -471,21 +471,26 @@ struct fuse_operations {
 	 * BIG NOTE: This is not equivalent to fsync().  It's not a
 	 * request to sync dirty data.
 	 *
-	 * Flush is called on each close() of a file descriptor.  So if a
-	 * filesystem wants to return write errors in close() and the file
-	 * has cached dirty data, this is a good place to write back data
-	 * and return any errors.  Since many applications ignore close()
-	 * errors this is not always useful.
+	 * Flush is called on each close() of a file descriptor, as opposed to
+	 * release which is called on the close of the last file descriptor for
+	 * a file.  One use of flush is to return errors from delayed writes.
+	 * However, such use is non-portable, because POSIX does not require
```

Comment:
I don't like the term "non-portable" - it sounds scary without giving any useful information. Instead, let's specifically say something like "Under Linux, errors returned by flush() will be passed to userspace as errors from close(), so flush() is a good place to write back any cached dirty data. However, many applications ignore errors on close(), and on non-Linux systems, close() may suceed even if flush() returns an error. For these reasons, [...]"